mittab/libs/outround_tab_logic/checks.py

```diff
@@ -0,0 +1,114 @@
+from mittab.apps.tab.models import *
+from mittab.libs.errors import PrevRoundNotEnteredError
+
+
+def lost_teams():
+    losers = [outround.loser.id
+              for outround in Outround.objects.all()
+              if outround.loser]
+
+    return losers
+
+
+def have_enough_judges_type(type_of_round):
+    lost_ids = [outround.loser.id
+                for outround in Outround.objects.all()
+                if outround.loser]
+
+    panel_size = 3
+
+    if type_of_round == BreakingTeam.VARSITY:
+        panel_size = TabSettings.get("var_panel_size", 3)
```

Comment:
Because this changes as the tournament goes on, I think it may be easiest to just not have this check and let tournaments validate for themselves. Maybe still display the info in the UI but don't use it for validation

Reply:
See above — the tab setting is still important as it determines how many slots are in the pairing view, I am assuming this is changed from out round to out round (and that totally works from my tests)